packages/app/source/client/controllers/todos-controller.js

```diff
@@ -6,13 +6,18 @@ Space.Object.extend(Todos, 'TodosController', {
   ],
 
   dependencies: {
-    configuration: 'configuration'
+    configuration: 'configuration',
+    todosStore: 'Todos.TodosStore',
   },
 
   eventSubscriptions() {
     return [{
       'Todos.TodoCreated': this._onTodoCreated,
       'Todos.TodoToggled': this._onTodoToggled,
```

Comment:
Yeah i also think that `toggle` is constraint to DOM / jQuery and just does not suit well as event (the receiver always needs to know the state) -> the only exception would be if you include all necessary information within the event, e.g: `isCompleted: true` property means "it was toggled from open to completed". But i guess it's easier to just have two dedicated UI events for this.